tensorflow.go

```diff
@@ -0,0 +1,172 @@
+package tensorflow
+
+import (
+	"encoding/binary"
+	"fmt"
+	"reflect"
+	"unsafe"
+
+	"github.com/golang/protobuf/proto"
+	tf "github.com/tensorflow/tensorflow/tensorflow/go"
+)
+
+type Session struct {
+	session tf.TF_Session
+}
+
+func NewSession() (*Session, error) {
```

Comment:
General comment:  Most of the functions provided here are missing doc-producing comments.  cf: https://github.com/golang/go/wiki/CodeReviewComments#doc-comments